BuildAndTest.cmd

```diff
@@ -37,11 +37,11 @@ echo         public const string Version = AssemblyVersion + Prerelease;
 echo     }                                                                           >> %VERSION_CONSTANTS%
 echo  }                                                                              >> %VERSION_CONSTANTS%
 
-%~dp0.nuget\NuGet.exe restore %~dp0src\BinSkim.sln -ConfigFile "%NuGetConfigFile%" -OutputDirectory "%NuGetPackageDir%"

```

Comment:
Not yet, we still use it for packing via a .nuspec.  I haven't looked into what's required to move us over to packing via 'dotnet pack' (which targets a .csproj file)--our packing needs are a little nonstandard (we're using it for full tool distribution instead of library distribution), so that may be slightly more challenging to port to the .csproj packing model.